collector/bgp.go

```diff
@@ -233,7 +231,7 @@ func processBGPSummary(ch chan<- prometheus.Metric, jsonBGPSum []byte, AFI strin
 							jsonDesc := struct{ Desc string }{}
 							if err := json.Unmarshal([]byte(d), &jsonDesc); err != nil {
 								// Don't return an error as unmarshalling is best effort.
-								logger.Error("cannot unmarshal bgp description", "description", peerDesc[vrfName].BGPNeighbors[peerIP].Desc, "err", err)
+								logger.Error("cannot unmarshal bgp description", "description", peerDesc[vrfName].BGPNeighbors[peerIP].Desc, "err", err.Error())
```

Comment:
It isn't necessary to invoke the `.Error()` method when passing the error as a value in a key-value pair. It is only necessary if the error is the log _message_ (i.e., first param), since that **must** be a string.
cf.
```go
	err := errors.New("foo error")
	slog.Error("aiiee", "err", err)
	slog.Error(err.Error())
```
```
2009/11/10 23:00:00 ERROR aiiee err="foo error"
2009/11/10 23:00:00 ERROR foo error
```